### ISSUE TITLE
Issue with LF4 on Precalculus Checkit

### ISSUE THREAD

**Issue by sharonakrinsky** [TeamBasedInquiryLearning/library#435]
Hi All!

There is a coding issue on LF4 for the Preview Edition of the Precalculus. When task 2 is set to ask for a perpendicular line it is not giving the correct answer. We were working with a copy of this problem in our own copy of CheckIt and we discovered that lines 45 and 47 of the code need to be switched (it is basing the choices off of how task 1 is set, but if task 1 is parallel then task 2 is perpendicular and vice versa). 

Basically, it is not doing the negative reciprocal of the slope for the perpendicular answer in task 2.

**Comment by siwelwerd** [comment]
@sharonakrinsky Thanks for the detailed report, that made this very easy to fix.
